sweetest/src/main/java/com/mysugr/sweetest/framework/cucumber/HookOrder.kt

```diff
@@ -3,7 +3,12 @@ package com.mysugr.sweetest.framework.cucumber
 object HookOrder {
 
     const val INITIALIZE_FRAMEWORK = 0
-    const val INITIALIZE_STEPS = 10000
-    const val INITIALIZE_DEPENDENCIES = 20000
-    const val SETUP = 30000
-}
\ No newline at end of file
+    const val INITIALIZE_STEPS = 10_000
```

Comment:
Just out of interest, where do this numbers come from?

Reply:
there are arbitraty order numbers, but they are important for ordering the Cucumber hooks (see [here](https://cucumber.io/docs/cucumber/api/#hooks)).